.azure_pipelines/performance_check/configs/bert.json

```diff
@@ -58,6 +58,7 @@
         "quantization": {
             "type": "OnnxQuantization",
             "config": {
+                "quant_preprocess": true,
```

Comment:
could we add this knowledge when generating auto-tune config?

Reply:
Yes~ We should. It is a good experience to run pre process for quantization.